### PR TITLE
 New Feature: Smash Box and Hit Box Support

### PR DESCRIPTION
**Pull request by Luberry**
I have designed hardware, and a version of the RetroSpy protocol that can be used for the Hit Box Arcade Smash Box and Hit Box using my board [here](https://gitlab.com/Luberry/smashboxledspy).

I have included a modified version of the RetroSpy driver to support this, as well as a couple of base themes.

### CODE REVIEW

**Comment by KernelZechs** [comment]
@Luberry congrats on being the first to do a PR, this is awesome. I'll review and see if I can get a release out sometime this week. I'll let you know if I find anything.

**Comment by KernelZechs** [comment]
@Luberry also thank you! This is an awesome feature you added. Thank you for your hard work. Your name will be added to the about panel with a link to your github.

File: app/themes/ojd-smash-box/images/hitboxSmashBoxUltimate.svg [hunk 1]
```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
[the remark this answers is not in the document]
Comment (Luberry) [reply app/themes/ojd-smash-box/images/hitboxSmashBoxUltimate.svg]: thats perfectly fine, I was just learning how to do everything here, and am interested to see how to change the centering and such. I'm more of a backend engineer, and haven't done much JS and such. this was a learning curve for me.

[the remark this answers is not in the document]
Comment (Luberry) [reply app/themes/ojd-smash-box/images/hitboxSmashBoxUltimate.svg]: also ill definitely help port it over to the react port. just let me know when you are ready

[the remark this answers is not in the document]
Comment (KernelZechs) [reply app/themes/ojd-smash-box/images/hitboxSmashBoxUltimate.svg]: No worries, you did fine! Thank you again for doing this, I'm going to try and get a release out this weekend and I'll send you the tweaks I did along with it.

**Pull request merged**